CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 ## 1.8.6 (Upcoming)
 
+### NOTES:
+* hcp: users of HCP for storing metadata for their builds will see some changes in
+       how names are displayed during a packer build for JSON templates. Previously
+       only the name or the type were displayed if both were available, and now both
+       will be displayed. This does not impact the behaviour of options such as only
+       or except, they will continue to work as they did before.
+* hcp: iteration fingerprints used to be computed from the Git SHA of the repository
```

Comment:
This a breaking change, right? So I think this should be highlighted somewhere

Reply:
I've added this to the 1.9.0 milestone, as it is a breaking change.